lib/form/html.js

```diff
@@ -137,8 +141,31 @@ const renderer = (type, errors) => {
         render += `<div id="${input.id}" class="form--field ${required}">`;
       }
 
-      // Add error messaging and required
-      render += addRequired(addError(html, input, errors), input);
+      // Wraps repeatable attributes around div
+      if (Array.isArray(input.inputs)) {
+        context.forEach((data, index) => {
+          render += `<div id="sub-${input.id}" class="form-repeatable">`;
+          html = nunjucks.renderString(input.html, data);
+
+          // Add error messaging and required
+          render += addRequired(addError(html, input, errors), input);
+          render += `<input type="button" value="delete" class="delete-button" id="${input.id}-del--${index}">`;
+          render += '</div>';
+        });
+        render += `<input type="button" value="add" class="add-button" id="${input.id}-add">`;
```

Comment:
`<button>`

note: added a [new issue for having the `add` text i18 compliant](https://github.com/punchcard-cms/content-types/issues/54) - does not need to be in this PR.